Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/MenuManager.cpp

```diff
@@ -587,10 +589,30 @@ namespace AzToolsFramework
                 "Menu Manager - Could not display menu \"%s\" - menu has not been registered.", menuIdentifier.c_str()));
         }
 
+        m_lastDisplayedMenuIdentifier = menuIdentifier;
         menuIterator->second.DisplayUnderCursor();
+
         return AZ::Success();
     }
 
+    MenuManagerPositionResult MenuManager::GetLastContextMenuPosition() const
+    {
+        if (m_lastDisplayedMenuIdentifier.empty())
+        {
+            return AZ::Failure(AZStd::string::format(
+                "Menu Manager - Could not return last context menu position. No menu was displayed yet."));
+        }
+
+        auto menuIterator = m_menus.find(m_lastDisplayedMenuIdentifier);
+        if (menuIterator == m_menus.end())
+        {
+            return AZ::Failure(
+                AZStd::string::format("Menu Manager - Could not return last context menu position. Menu could not be found."));
```

Comment:
Same comment as above
```suggestion
                AZStd::string("Menu Manager - Could not return last context menu position. Menu could not be found."));
```